### PR TITLE
Update microsoft-teams.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**
updated steps and screenshots

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/microsoft-teams.md [hunk 1]
```diff
@@ -43,35 +43,37 @@ This will generate a webhook URL which can then be used to post messages to this
 
 ## Configuring Microsoft Teams
 
-1. In the [Integrations](https://go.postman.co/workspaces) tab for your workspace, select Microsoft Teams from the list of third party integrations.
+1. From the [Integrations](go.postman.co/integrations/browse?category=all) search page, search and select Microsoft Teams from the results.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/microsoft-teams.md:46]: I think we can remove the 1. here as you removed the below numbers

File: src/pages/docs/integrations/available-integrations/microsoft-teams.md [hunk 2]
```diff
@@ -43,35 +43,37 @@ This will generate a webhook URL which can then be used to post messages to this
...
 ### Add a team activity feed to Microsoft Teams
 
 To add a team activity feed to Microsoft Teams:
 
-1. Click **Add Integration**.
-1. Give your integration a nickname and provide your [incoming webhook URL](#create-a-microsoft-teams-webhook-url) to receive team updates in Microsoft Teams.
+Select **Add Integration**.
 
-   <img alt="Import from github" src="https://assets.postman.com/postman-docs/add-team-activity-feed.jpg" width="350px"/>
+Give your integration a nickname and provide your [incoming webhook URL](#create-a-microsoft-teams-webhook-url) to receive team updates in Microsoft Teams.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/microsoft-teams.md:56]: Maybe we can specify the differing names - "..provide your [incoming webhook URL]() as the **Notification URL** to receive.."

File: src/pages/docs/integrations/available-integrations/microsoft-teams.md [hunk 3]
```diff
@@ -43,35 +43,37 @@ This will generate a webhook URL which can then be used to post messages to this
...
 
 To send monitor run results to Microsoft Teams:
 
-1. Click **Add Integration**.
-1. In the **Monitor Run Results** page, select the monitor whose results you want to send to Microsoft Teams, and enter the notification URL.
+Select **Add Integration**.
+
+In the **Monitor Run Results** page, select the monitor whose results you want to send to Microsoft Teams, and enter the notification URL.
 
    [![ms_teams monRun](https://assets.postman.com/postman-docs/WS-integration-msTeams-monRun.png)](https://assets.postman.com/postman-docs/WS-integration-msTeams-monRun.png)
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/microsoft-teams.md:72]: Can we update this one? I think cropping to the form and adding a width would make the text a little larger

File: src/pages/docs/integrations/available-integrations/microsoft-teams.md [hunk 4]
```diff
@@ -43,35 +43,37 @@ This will generate a webhook URL which can then be used to post messages to this
...
 
-1. Click the **Add Integration** button to see your monitors in the "Configured Integrations" view. You can also click the "Advanced Options" link to indicate if you want notifications when all monitor runs are completed, or if you want notifications for three monitor run failures and then the first successful monitor run.
+Select the **Add Integration** button to see your monitors in the "Configured Integrations" view. You can also click the "Advanced Options" link to indicate if you want notifications when all monitor runs are completed, or if you want notifications for three monitor run failures and then the first successful monitor run.
```
Comment (claireriley) [inline src/pages/docs/integrations/available-integrations/microsoft-teams.md:74]: "Select the **Add Integration** button" -> "Select **Add Integration**"

**Review by claireriley — CHANGES_REQUESTED** [review]
Looks great @jerry-li-dev ! Few small suggestions

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Review by claireriley — APPROVED** [review]
LGTM

**Pull request merged**